contracts/asset-proxy/contracts/src/bridges/KyberBridge.sol

```diff
@@ -0,0 +1,160 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
+import "@0x/contracts-erc20/contracts/src/interfaces/IEtherToken.sol";
+import "@0x/contracts-erc20/contracts/src/LibERC20Token.sol";
+import "@0x/contracts-exchange-libs/contracts/src/IWallet.sol";
+import "../interfaces/IERC20Bridge.sol";
+import "../interfaces/IKyberNetworkProxy.sol";
+
+
+// solhint-disable space-after-comma
+contract KyberBridge is
+    IERC20Bridge,
+    IWallet
+{
+    // @dev Structure used internally to get around stack limits.
+    struct TradeState {
+        IKyberNetworkProxy kyber;
+        IEtherToken weth;
+        address fromTokenAddress;
+        uint256 fromTokenBalance;
+        uint256 payableAmount;
+        uint256 minConversionRate;
+    }
+
+    /// @dev Address of the WETH contract.
```

Comment:
Optional: It would be cool to use the `DeploymentConstants` pattern here.

Reply:
Should we have a canonical `DeploymentConstants` that we can use if we're going to use this pattern more frequently? Unused internal functions shouldn't affect the bytecode of the contracts, right?